fetch.bs

```diff
@@ -5352,8 +5352,20 @@ run these steps:
  <li><p>Invoke <a>set <var>request</var>'s referrer policy on redirect</a> on <var>request</var> and
  <var>actualResponse</var>. [[!REFERRER]]
 
+ <li><p>Let <var>recursive</var> be true.
+
+ <li>
+  <p>If <var>request</var>'s <a for=request>redirect mode</a> is "<code>manual</code>", then:
+
+  <ol>
+   <li><p>Assert: <var>request</var>'a <a for=request>mode</a> is "<code>navigate</code>".
```

Comment:
This is incorrect. You can always set it to "manual" and get an opaque response at the moment. Mainly useful in service workers with navigations though.

I suggest we do something like

> Let recursive be true if ....; otherwise false.

instead.

Reply:
Yea but if it's manual non-navigation you wouldn't get to this bit as fetch would never process your redirect. Anyway, revised.

Reply:
Wait, that's good pushback and solid rationale. Why revise?

Reply:
Mainly because that assert is potentially more confusing than helpful and I wasn't sure about it. Should we keep it or throw it? I could go either way.

Reply:
Done